### PR TITLE
[ADD] l10n_qa: Add localization package for qatar

### PR DESCRIPTION
**Pull request by MuhamedAbdallaMuhamed**
[ADD] l10n_qa: Add localization package for qatar

Add CoA and account groups with all the default account mappings for qatar

Improve user experience in qatar

task-id#3927918

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_qa/i18n/qa_QA.po [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Translation of Odoo Server.
```
Comment (alialfie) [inline addons/l10n_qa/i18n/qa_QA.po:1]: I'm not sure of the name of this file. According to [this](https://www.localeplanet.com/icu/iso639.html#ar), `ar_001` is standard Arabic which I think is the one we should use while the one for Qatar is `ar_QA`. What is `qa_QA` supposed to be?

**Review by alialfie — COMMENTED** [review]
Thanks @muab-odoo! Just a few questions :)

File: addons/l10n_qa/demo/demo_company.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="partner_demo_company_qa" model="res.partner">
+        <field name="name">QA Company</field>
+        <field name="vat"/>
```
[the remark this answers is not in the document]
Comment (muab-odoo) [reply addons/l10n_qa/demo/demo_company.xml]: No, we can add one.

[the remark this answers is not in the document]
Comment (muab-odoo) [reply addons/l10n_qa/demo/demo_company.xml]: Sorry Ali, I was mistaken here.
There is no vat yes according to the PO :sweat_smile:

**Review by alialfie — APPROVED** [review]
Thanks for the quick update! My only last remark is we should include the ar_001 translation instead of the ar_QA one, as this is the standard Arabic translation and it's likely the one that will be used. If needed, the ar_QA one can be added too but let's at least include the ar_001 one. I will already approve this as it's a small change. Good job!

LGTM @FlorianGilbert

**Comment by muab-odoo** [comment]
@FlorianGilbert 
Needed for v18 ^-^

**Review by FlorianGilbert — APPROVED** [review]
LGTM

@robodoo r+

**Pull request closed without merging**